01-js-intro-and-jquery/spotify/js/myscripts.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function(){
+   console.log("la respueta de la fucking console.log es: " + $.post('https://api.spotify.com/v1/search?q=Rolling%20Stones&type=album')
```

Comment:
You are missing a fucking ")" here :).
Anyway you can't just log the $.post method, try to use a callback (http://api.jquery.com/jquery.post/).